README.md

```diff
@@ -11,13 +11,36 @@ These instructions will get a copy of the project up and running on your local m
 ### Local setup
 
 1. `git clone https://github.com/TechforgoodCAST/fusebox-insights.git`
-2. `cd fusebox_insights`
+2. `cd fusebox-insights`
 3. `bundle install`
 4. `yarn install`
+
+### Set up database
+
+Do either of the below
+
+#### Create a new database
 5. `rails db:setup`
-7. `rails s` to start local development server
+You'll need to update `database.yml` with your local Postgre credentials.
+
+#### Clone the existing database
```

Comment:
I'd suggest not including these instructions here as they are specific to how the app is currently deployed and perhaps something we want to keep private.